hvantk/htables/expression_atlas.py

```
@@ -0,0 +1,286 @@
+        sdrf_df = pd.read_csv(sdrf_file, **read_params)
+
+        # Clean column names
+        sdrf_df.columns = sdrf_df.columns.str.strip()
+
+        # Drop unused columns
+        sdrf_df.drop(columns=['unused'], inplace=True, errors='ignore')
+
+        return sdrf_df
+
+    except FileNotFoundError:
+        raise FileNotFoundError(f"SDRF file not found: {sdrf_file}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)